nest/src/services/client.module.ts

```diff
@@ -0,0 +1,13 @@
+import {ClientsModule, RmqOptions, Transport} from '@nestjs/microservices';
+import {ConfigModule, ConfigService} from '@nestjs/config';
+import {API_SERVICE} from '../constants/client.constants';
+
+export const ClientModule = ClientsModule.registerAsync([{
```

Comment:
Shouldn't this be called `ApiClientModule`?

Reply:
No, as it holds an array of clients.